### ISSUE TITLE
[ActivityItem-Screen reader] Narrator focus and Keyboard focus is not synchronized when Philippe Lampros control selected. 

### ISSUE THREAD

**Issue by v-doku** [OfficeDev/office-ui-fabric-react#9638]
[ActivityItem-Screen reader] Narrator focus and Keyboard focus is not synchronized when Philippe Lampros control selected. 

Actual Result:Narrator focus and Keyboard focus is not synchronized when Philippe Lampros control is clicked and dialog box opens
Expected Result :Narrator focus and Keyboard focus should be synchronized when Philippe Lampros control is clicked and dialog box opens

Detailed description of this issue can be found here [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/Documents/Fabric%20Control/Fabric%20Controls%202nd%20Phase/ActivityItem/Fabric-%20Activityitem.xlsx?d=w42e361b637854a43997d752301d890be&csf=1&e=kLKtQU)

**Comment by khmakoto** [comment]
Hi @v-doku, it seems that I do not have permission to access your link and thus cannot check your issue. Can you please provide a link that I have access to or request to give me permission to access that link? Thanks!

**Comment by v-doku** [comment]
Provided the access for share point link. Sorry for inconvenience

**Comment by khmakoto** [comment]
Hi @v-doku, this is a bug with Edge's alert and not our controls. I'll be closing this down as external since it is not a bug with Fabric.
